test/distributed/org/apache/cassandra/distributed/test/accord/AccordLoadTest.java

```diff
@@ -43,20 +55,35 @@ public class AccordLoadTest extends AccordTestBase
     @BeforeClass
     public static void setUp() throws IOException
     {
-        AccordTestBase.setupCluster(builder -> builder.withConfig(config -> config.set("lwt_strategy", "accord").set("non_serial_write_strategy", "accord")), 2);
+        CassandraRelevantProperties.SIMULATOR_STARTED.setString(Long.toString(MILLISECONDS.toSeconds(currentTimeMillis())));
```

Comment:
why are you setting this?  we don't setup simulator for jvm-dtest

Reply:
the simulator log file is the most suitable for running this, and this exists only to setup logging for the instances for that file